### PR TITLE
GITC-632 Have added hardcoded mini-banner for Gitcoin Building Gitcoi…

### PR DESCRIPTION
**Pull request by nutrina**
…n. This will be shown instead of an active item CLR item if the name is 'Gitcoin Building Gitcoin'

##### Description

Have added hardcoded banner for Gitcoin Building Gitcoin 

##### Refers/Fixes

GITC-632

##### Testing

These are screenshots for my local instance:
image

image

### CODE REVIEW

File: app/grants/templates/grants/landing/round_by_type.html [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+<div class="col-12 col-lg-6">
+  <a
+    href="https://grants.gtcdao.net/"
+    class="active-clr-banner d-block text-decoration-none position-relative mt-3"
+  >
+    <img alt="Gitcoin Building Gitcoin"  src="{% static 'landingpage/gr12_small_banner.png' %}" class="w-100">
+  </a>  
+</div>
+
+{% else %}
```
Comment (thelostone-mc) [inline app/grants/templates/grants/landing/round_by_type.html:15]: @nutrina i don't think we'll have an active_round called `Gitcoin Building Gitcoin`
I'd say just throw it below https://github.com/gitcoinco/web/blob/master/app/grants/templates/grants/landing/active_clr_round.html#L22

Comment (nutrina) [reply app/grants/templates/grants/landing/round_by_type.html]: @thelostone-mc Have moved the code as you instructed. But it does not really look good in mobile. Will continue to tweak it a bit ...

Comment (thelostone-mc) [reply app/grants/templates/grants/landing/round_by_type.html]: Haha alright 🙌

Comment (nutrina) [reply app/grants/templates/grants/landing/round_by_type.html]: @nutrina Should be final (see the screenshots). It is not perfect, but better than it was.

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**